test/test.urlshortener.v1.ts

```diff
@@ -1,102 +0,0 @@
-// Copyright 2013-2016, Google, Inc.
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//    http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-import * as assert from 'assert';
-import * as nock from 'nock';
-import * as path from 'path';
-import {GoogleApis, urlshortener_v1} from '../src';
-import {Utils} from './utils';
-
-async function testSingleRequest(urlshortener: urlshortener_v1.Urlshortener) {
```

Comment:
Just to double check - were these tests verifying the specific behaviors of url shortener, or were they covering broad features that aren't tracked elsewhere?  I know it's somewhat unfair to ask that, since I don't even know 👅

Reply:
they seem fairly urlshortner specific, in fact there's a fixture I should also remove.